### PR TITLE
Postman

### PR DESCRIPTION
**Pull request by v-krishnag**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: postman-collection/IoT Central.postman_collection.json [hunk 1]
```diff
@@ -2745,6 +2745,191 @@
...
 					},
 					"response": []
+				},
+				{
+					"name": "Delete a scheduled job by Id Copy",
+					"request": {
+						"method": "DELETE",
+						"header": [],
+						"url": {
+							"raw": "https://{{yourAppSubdomain}}.azureiotcentral.com/api/scheduledJobs/{{scheduledJobId}}?api-version=2022-07-31",
```
Comment (dominicbetts) [inline postman-collection/IoT Central.postman_collection.json]: Shouldn't be hardcoding the api version.

File: postman-collection/IoT Central.postman_collection.json [hunk 2]
```diff
@@ -2745,6 +2745,191 @@
...
+							"mode": "raw",
+							"raw": "{\r\n    \"displayName\": \"New Scheduled Job\",\r\n    \"group\": \"6fecf96f-a26c-49ed-8076-6960f8efba31\",\r\n    \"data\": [\r\n        {\r\n            \"type\": \"cloudProperty\",\r\n            \"target\": \"dtmi:azurertos:devkit:hlby5jgib2o\",\r\n            \"path\": \"Company\",\r\n            \"value\": \"Contoso\"\r\n        }\r\n    ],\r\n    \"schedule\": {\r\n        \"start\": \"2022-10-24T22:29:01Z\",\r\n        \"recurrence\": \"daily\"\r\n    }\r\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "https://{{yourAppSubdomain}}.azureiotcentral.com/api/jobs/{{scheduledJobId}}?api-version={{dataPlanePreview}}",
```
Comment (dominicbetts) [inline postman-collection/IoT Central.postman_collection.json]: I think schedules jobs are GA, not preview. Here and elsewhere.

**Review by dominicbetts — CHANGES_REQUESTED** [review]
A couple of changes please.

**Review by dominicbetts — CHANGES_REQUESTED** [review]
REviewing now

**Comment by v-krishnag** [comment]
> I don't see the enrollment group calls in here - can you check???


Hi Dominic, this commit https://github.com/Azure-Samples/iot-central-docs-samples/pull/9/commits/3695743681c93a8d23112756bebcac12e1900cd4, has the enrollment group changes

File: postman-collection/IoT Central.postman_collection.json [hunk 3]
```diff
@@ -2014,6 +2014,366 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "Add a enrollment group",
```
Comment (v-krishnag) [inline postman-collection/IoT Central.postman_collection.json]: Hi @dominicbetts, I added the changes here for the enrollment groups

Comment (dominicbetts) [reply postman-collection/IoT Central.postman_collection.json]: Sorry - just figured out where the changes are.

File: postman-collection/IoT Central.postman_collection.json [hunk 4]
```diff
@@ -2014,6 +2014,366 @@
...
+									"key": "api-version",
+									"value": "{{dataPlaneGA}}"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Add a X509 certificate to a enrollment group",
```
Comment (dominicbetts) [inline postman-collection/IoT Central.postman_collection.json]: "an X.509" - here and elsewhere

File: postman-collection/IoT Central.postman_collection.json [hunk 5]
```diff
@@ -2014,6 +2014,366 @@
...
+					"response": []
+				},
+				{
+					"name": "Verify X509 certificate of an enrollment group",
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n  \"certificate\": \"{{base64-certificate}}\"\r\n}",
```
Comment (dominicbetts) [inline postman-collection/IoT Central.postman_collection.json]: You need to have a separate environment variable for the verification cert - so two environment variables in the collection to hold two different base-64 strings.

File: postman-collection/IoT Central.postman_collection.json [hunk 6]
```diff
@@ -2014,6 +2014,366 @@
...
+									"key": "api-version",
+									"value": "{{dataPlaneGA}}"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get a enrollment group",
```
Comment (dominicbetts) [inline postman-collection/IoT Central.postman_collection.json]: "an enrollment group" - here and elsewhere.

**Review by dominicbetts — CHANGES_REQUESTED** [review]
Can you add examples of the API calls to create symmetric key enrollment groups.

File: postman-collection/IoT Central.postman_collection.json [hunk 7]
```diff
@@ -1837,37 +1837,387 @@
...
+				},
+				{
+					"name": "Add an X.509 enrollment group",
+					"request": {
+						"method": "PUT",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n  \"displayName\": \"My group\",\r\n  \"enabled\": true,\r\n  \"type\": \"iot\",\r\n  \"attestation\": {\r\n    \"type\": \"x509\"\r\n  }\r\n}\r\n",
+							"options": {
```
Comment (dominicbetts) [inline postman-collection/IoT Central.postman_collection.json:2026]: Can we have a different display name for the two enrollment groups.

**Review by dominicbetts — CHANGES_REQUESTED** [review]

**Pull request merged**